tasks/45_equal-sides/equal-sides.js

```diff
@@ -0,0 +1,14 @@
+function findEqualIndex(arr) {
+  let leftSum = 0;
+  let rightSum = arr.reduce((sum, item) => sum + item);
+  return arr.reduce((sum, item, index) => {
+    rightSum -= item;
+    if (leftSum === rightSum && sum === -1) {
+      return index;
+    }
+    leftSum += item;
+    return sum;
```

Comment:
`sum` is not being changed from iteration to iteration. What is reason of using .`reduce()` then?

Reply:
It's changed if `leftSum === rightSum && sum === -1` is truthy and instead of `sum` `index` is returned.